src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleInventoryCleaner.kt

```diff
@@ -145,7 +146,7 @@ object ModuleInventoryCleaner : Module("InventoryCleaner", Category.PLAYER) {
                         ) != true) && weightedItem.slot != hotbarSlotToFill.second
                     ) {
                         if (executeAction(weightedItem.slot, hotbarSlotToFill.second, SlotActionType.SWAP)) {
-                            wait(inventoryConstraints.delay.random())
+                            wait { inventoryConstraints.delay.random() }
```

Comment:
This might have changed the way wait works... { ... } instead of ticks as argument.